loops.js

```diff
@@ -18,8 +21,22 @@
 // klaar!
  // ==========================================
 
+// for (let j = 0; j < 5; j++) {
+//     if (j === 4) {
+//         console.log("Klaar!");
+//     } else {
+//         console.log("loop...");
+//     }
+// }
 
-
+for (let j = 0; j < 5; j++) {
+    if (j <= 3) {
+        console.log("Loop...");
+    } else {
+        console.log("Klaar");
+    }
+}
```

Comment:
Als er een specifieke reden is dat je 'j' gebruikt i.p.v. 'i' in de for loop, heb ik niks gezegd.
Anders is het volgens mij gebruikelijk om 'i' te gebruiken.

Reply:
Ik gebruikte een andere letter omdat ik dacht dat je dat niet zo vaak kan gebruiken in hetzelfde bestand, omdat die i dan al gebruikt wordt. Dat moet bij Java in de Backend lijn namelijk wel haha! Maar met de 'scope' hier in javascript kan je dus gewoon steeds i gebruiken. Goed dat je het benoemd, nu heb ik dat ook door :))